### DISCUSSION TITLE
TypeScript implementation.

### DISCUSSION THREAD

**Discussion by Andres10121997** [twbs/bootstrap#33623] [Ideas]
[bootstrap.zip](https://github.com/twbs/bootstrap/files/6306529/bootstrap.zip)

Hello, how are you?
For bootstrap, haven't you thought about using TypeScript?
I tried to attach a zip with the implementation of this language, but I don't know if I made it.

**Comment by alegorico** [comment]
Hello,
 I don't know if there has been interest on this topic, but at least for the components, it would be interesting to think about this challenge. Is there a work team for this initiative?
Regards.
